src/Microsoft.Azure.SignalR.Management/ServiceManagerOptions.cs

```diff
@@ -22,6 +21,27 @@ public class ServiceManagerOptions
         /// </summary>
         public string ConnectionString { get; set; } = null;
```

Comment:
If a user sets a service endpoint and then sets a connection string, then the connection string does not take effect, which is not excepted. I think setting service endpoint each time setting connection string is better?